src/Disruptor/Util/Constants.cs

```diff
@@ -5,5 +5,6 @@ namespace Disruptor
     internal static class Constants
     {
         internal const MethodImplOptions AggressiveOptimization = (MethodImplOptions)512;
+        internal const int CacheLineSize = 128;
```

Comment:
The cache line size is 64, maybe this field could be renamed CacheLinePadding?

Reply:
Naming things... On some Samsung ARM CPUs cache line is 128 already, and Apple M1 has 128 reportedly https://news.ycombinator.com/item?id=25660467

But CacheLinePadding is a good name.